shell/app/layout/pages/page-container/components/sidebar.tsx

```diff
@@ -149,6 +149,10 @@ const SideBar = () => {
     },
   ].filter(a => a.show);
 
+  useMount(() => {
+    orgStore.effects.getJoinedOrgs();
```

Comment:
this effects only need to be executed once,  add a exist  condition to forbidden unnecessary request